docs/src/docs/next-steps.md

```diff
@@ -9,23 +9,49 @@ In this guide you'll learn how to:
 ## Making your own Kitfile
 
 A Kitfile is the configuration document for your ModelKit. It's written in YAML so it's easy to read. There are four main parts to a Kitfile:
+
 1. The `package` section: Metadata about the ModelKit, including the author, description, and license
-1. The `code` section: Path to any codebases related to the project, including Jupyter notebook folders
-1. The `datasets` section: Path to any datasets
-1. The `model` section: Path to the serialized model
+1. The `code` section: Path and information about codebases related to the project, including Jupyter notebook folders
+1. The `datasets` section: Path and infromation on included datasets
+1. The `model` section: Path and information on the serialized model
 
 A Kitfile only needs the `package` section, plus one of the other sections.
 
-The `datasets` and `code` sections can include multiple assets, but the `model` section can only contain a single model (you can chain models by using multiple ModelKits).
+The `package` and `model` sections can only contain a single model (you can chain models by using multiple ModelKits).
+
+The `datasets` and `code` sections can include multiple assets. Each asset is indicated by starting with a dash. The dash is required even if you are only packaging a single dataset or codebase.
 
-Here's an example Kitfile:
+Here's a snippet of a KitFile that contains two datasets, notice that each starts with "-":
 
 ```yaml
-manifestVersion: v1.0.0
+datasets:
+- description: Forum postings from photo sites
+  name: training data
+  path: ./data/forum-to-2023-train.csv
+  
+- name: validation data
+  path: ./data/test.csv
+```
+
+...and here's an example of a single codebase, it still needs to start with a "-":
+
+```yaml
+code:
+- description: Jupyter notebook with model training code in Python
+  path: ./notebooks
+```
+
+Any relative paths defined within the Kitfile are interpreted as being relative to the context directory sent to the `kit pack` command.
+
+### Kitfile Examples
+
+Here's a complete Kitfile example, with a model, two datasets, and a codebase:
+
+```yaml
+manifestVersion: 1.0
 
 package:
-  authors:
-  - Jozu
+  authors: Jozu
```

Comment:
Authors is also a yaml list, so the formats are
```yaml
authors:
  - Jozu
```
or
```yaml
authors: ["Jozu"]
```

Reply:
Thanks, fixed in latest commit